src/token.ts

```diff
@@ -39,7 +39,7 @@ function getAccountBalanceForTokenContractAddress(
   )
 }
 
-function getHistoricAccountBalanceForTokenContractAddress(
+function getHistoricialAccountBalanceForTokenContractAddress (
```

Comment:
It seems you missed the rename in the `export default` section

Reply:
While reviewing it I also noticed that I misspelled the `Historicial` word!! it should be `getHistoricalAccountBalanceForTokenContractAddress`

Sorry about that!

Reply:
No problem :) Fixing it rn